scripts/install.sh

```diff
@@ -58,24 +57,19 @@ download_binary() {
     local _router="$_dir/router$_ext"
 
     local _release_download_url="$GITHUB_REPO/releases"
+    local _router_version=$DOWNLOAD_VERSION
     if [ "$DOWNLOAD_VERSION" == "latest" ]; then
-      # Github should return the latest release first.
-      parser=".[0].assets | map(select(.name | contains(\"$ARG_ARCH\")))[0]"
-    else
-      parser=". | map(select(.tag_name == \"$DOWNLOAD_VERSION\"))[0].assets | map(select(.name | contains(\"$ARG_ARCH\")))[0]"
+        local _response=$(curl -Ls -o /dev/null -w %{url_effective} $GITHUB_REPO/releases/latest)
+        _router_version=$(echo "$_response" | cut -d'/' -f 8)
+        [ "$_router_version" ] || { echo "Error: Failed to get asset version for '$ARG_ARCH', response: $_response" | awk 'length($0)<100' >&2; exit 1; }
     fi;
 
     say "Downloading release info for '$_release_download_url'" 1>&2
-    local _response=$(curl -s $_release_download_url)
-    #echo $_response | jq
-
-    local _id=$(echo "$_response" | jq "${parser}.id")
-    [ "$_id" ] || { echo "Error: Failed to get asset id for '$ARG_ARCH', response: $_response" | awk 'length($0)<100' >&2; exit 1; }
 
-    local _name=$(echo "$_response" | jq "${parser}.name")
-    [ "$_name" ] || { echo "Error: Failed to get asset name, response: $_response" | awk 'length($0)<100' >&2; exit 1; }
+    # Cut the 'v' prefix
+    local _name="router-$(echo "$_router_version" | cut -c2-)$_ext-$ARG_ARCH.tar.gz"
```

Comment:
If you want to avoid more cuts, I think most shells support this syntax:
```suggestion
    # Cut the 'v' prefix
    local _name="router-${_router_version:1}$_ext-$ARG_ARCH.tar.gz"
```
Feel free to ignore this suggestion.

Reply:
It's not working when I try to | sh